lib/wibox/widget/imagebox.lua

```diff
@@ -6,29 +6,49 @@
 -- @widgetmod wibox.widget.imagebox
 ---------------------------------------------------------------------------
 
+local lgi = require("lgi")
+local Rsvg = lgi.Rsvg
+local cairo = lgi.cairo
+
 local base = require("wibox.widget.base")
 local surface = require("gears.surface")
 local gtable = require("gears.table")
+local gdebug = require("gears.debug")
 local setmetatable = setmetatable
 local type = type
-local print = print
+local math = math
+
 local unpack = unpack or table.unpack -- luacheck: globals unpack (compatibility with Lua 5.1)
 
 local imagebox = { mt = {} }
 
+---Load rsvg handle form image file
+---@tparam string file Path to svg file.
+---@return Rsvg handle
+---@treturn number Image default width
+---@treturn number Image default height
+local function try_to_load_svg(file)
+    if not (Rsvg and file:match("%.svg$")) then return end
```

Comment:
I'm not sure that matching on the file extension is a good idea and the check for `Rsvg` is dead code: This will always be true-ish (see https://github.com/awesomeWM/awesome/pull/2779/commits/baabe51eef528440da0860a4dd2feae9fb296a8f#r291985379 )